moto/core/exceptions.py

```diff
@@ -53,22 +53,22 @@ class RESTError(HTTPException):
         "single_error": SINGLE_ERROR_RESPONSE,
         "wrapped_single_error": WRAPPED_SINGLE_ERROR_RESPONSE,
         "error": ERROR_RESPONSE,
-        "error_json": ERROR_JSON_RESPONSE,
```

Comment:
Can we remove the `ERROR_JSON_RESPONSE`-global, or is that still in use somewhere else?

Reply:
Yeah actually thought I had deleted this already. :thinking:  Removed this now and I've also changed the exception rendering for managedblockchain in the same way.